lib/stripe/payment_methods/source.ex

```diff
@@ -10,6 +10,8 @@ defmodule Stripe.Source do
 
   @type source_type :: String.t()
 
+  @type customer :: String.t()
```

Comment:
Could we change the type to `Stripe.id()`?  Effectively is `String.t()`, but a bit more expressive.

Reply:
Sure 👍

Reply:
Oops, wasn't paying attention. Fixed now